src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/SyntaxNodeUtils.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.AspNetCore.Razor.Language;
+using Microsoft.AspNetCore.Razor.Language.Syntax;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer;
+
+internal static class SyntaxNodeUtils
```

Comment:
I discovered yesterday that we have [SyntaxNodeExtensions](https://github.com/dotnet/razor/blob/main/src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Extensions/SyntaxNodeExtensions.cs) which this could go in, or if we keep it seperate, perhaps we should adopt naming similar to Roslyn and use `RazorSyntaxFacts` or something.

Reply:
Moved